mitodl/bootcamp-ecommerce
# Block/UnBlock list of users for getting certificates.

#### What are the relevant tickets?
fixes: #1135 

#### What's this PR do?
As a bootcamp admin, I often have a list of enrolled bootcampers who shouldn't get a certificate (typically because they dropped out before the bootcamp finished).

#### How should this be manually tested?
Execute the command in following format. 
```
python manage.py manage_certificates --run={RUN} --block camper1@example.com camper2@example.com camper3@example.com
python manage.py manage_certificates --generate-batch --run={RUN}
python manage.py manage_certificates --generate --run={RUN} --user={USER}
```
Certificate should not be generated for the blocked users in ~any~ provided enrollment run. 
But after, 
```
python manage.py manage_certificates --run={RUN} --unblock camper1@example.com camper2@example.com camper3@example.com
python manage.py manage_certificates --generate-batch --run={RUN}
python manage.py manage_certificates --generate --run={RUN} --user={USER}
```
Certificate should not be generated for the users. 

~#### NOTE: Currently executing the command with option `--block [USER LIST]` will block the users for getting certificate in ~any~ specific enrollment run. We are not targeting to block user for a specific enrollment run in the command.~
